ecs_fargate/README.md

```diff
@@ -8,65 +8,92 @@ Get metrics from all your containers running in ECS Fargate:
 * I/O metrics
 
 ## Setup
-### Installation
-
-The ECS Fargate check is packaged with the Agent, [run the Agent][1] with your containers to start collecting metrics.
-
-### Deploying the Datadog Agent on AWS Fargate
+The following steps cover setup of the Datadog Container Agent within AWS ECS Fargate. **Note**: Datadog Agent version 6.1.1 or higher is needed to take full advantage of the Fargate integration.
 
-To start monitoring AWS Fargate tasks and services, deploy the containerized Datadog Agent on Fargate. In addition to passing the usual `DD_API_KEY` environment variable, you must set the `ECS_FARGATE `environment variable to `true`.
+### Installation
+To monitor your ECS Fargate containers with Datadog, run the Agent as a container in same task definition as your application. To collect metrics with Datadog, each task definition requires a Datadog Agent container. Follow these setup steps:
```

Comment:
I believe we want to monitor tasks here not containers.

Reply:
" each task definition should include a sidecar Datadog Agent container. Follow these setup steps:"